src/index/base.cpp

```diff
@@ -354,49 +421,6 @@ bool BaseIndex::Commit(const CBlockLocator& locator)
     return true;
 }
 
-bool BaseIndex::Rewind(const CBlockIndex* current_tip, const CBlockIndex* new_tip)
```

Comment:
In 0550d174:

`BaseIndex::Rewind` definition was removed but not its declaration in the header.
Also, there is a comment in `IgnoreBlockConnected()` referring to this method that should be updated.